media/repository.html.ejs

```diff
@@ -13,86 +13,93 @@
   <div id="root">
     <div class="page">
       <div class="header" x-memo>
-        <h1 class="title">Create Project</h1>
-        <p class="subTitle">
+        <div class="title">Create Project</div>
```

Comment:
Any reason for going from `h1` to `div`? While this is not a standard "web"site I'd still argue that having proper semantics (e.g., using an `h1` for a first-level heading) is good.